modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,117 @@
+** Manifest files: Track data files and contain metadata about those files, such as record count, partition membership, and file paths.
+** Manifest list: Tracks all the manifest files belonging to a table, including file paths and upper and lower bounds for partition fields. 
+** Metadata file: Stores metadata about the table, including its schema, partition information, and snapshots. Whenever a change is made to the table, a new metadata file is created and becomes the latest version of the metadata in the catalog.
+--
++
+In the Redpanda Iceberg integration, the manifest files are in JSON format.
+* Catalog: Contains the current metadata pointer for the table. Clients reading and writing data to the table see the same version of the current state of the table. Redpanda creates the catalog in JSON format.
+
+When you enable the Iceberg integration for a Redpanda topic, Redpanda brokers store streaming data in the Iceberg-compatible format in Parquet files in object storage, in addition to writing Kafka messages stored in segment files on disk. Storing the streaming data in Iceberg tables in the cloud allows you to derive real-time insights through business intelligence and machine learning tools that work with Iceberg. 
+
+
+== Enable Iceberg integration
+
+To use the Iceberg integration for a topic, you must set Redpanda cluster configuration `enable_datalake` to true, and also explicitly set the topic configuration to `true`.
+
+. Enable the Iceberg integration for the cluster by setting the cluster configuration `enable_datalake` to `true`. 
++
+[,bash]
+----
+rpk cluster config set enable_datalake true 
+----
++
+[,bash,.no-copy]
+----
+Successfully updated configuration. New configuration version is 2.
+----
+
+. Create a new topic.
++
+[,bash,]
+----
+rpk topic create <new-topic> --partitions 1 --replicas 1
+----
++
+[,bash,.no-copy]
+----
+TOPIC       STATUS
+new-topic   OK
+----
+
+. Enable the integration for the topic. 
++
+[,bash]
+----
+rpk topic alter-config <new-topic> --set iceberg_topic=true 
+----
++
+[,bash,.no-copy]
+----
+TOPIC       STATUS
+new-topic   OK
+----
+
+. Optional: Register a schema for the topic. Redpanda uses the schema to derive the Iceberg table format and write data files to object storage in this format. Protobuf schemas are supported.
++
+If you don't configure a schema for the topic, Redpanda by default uses a simple schema consisting of the record’s key, value, and the original message timestamp. If you don't have a need for a strict schema and can use the data in a more semi-structured format, consider the schemaless approach.
++
+[,bash]
+----
+rpk registry schema create <new-topic> --schema </path-to-schema> --type <format>
+----
++
+[,bash,.no-copy]
+----
+SUBJECT     VERSION   ID   TYPE
+new-topic   1         1    PROTOBUF
+----
+
+As you produce messages to the topic, the data also becomes available to consume from object storage by Iceberg-compliant clients.
+
+== Consume data in Iceberg tables
```

Comment:
Should we just repurpose the same Clickhouse example (with the schemaless option) from our demos?

Reply:
I think it would be great to show one schemaless and one with a schema.  Reach out to Andrew Wong to ask for a simple example with a schema.

Reply:
it would be really nice to have:

- 1 example consuming data via a system using an external REST catalog (e.g. Polaris catalog in Snowflake) by just doing SHOW TABLES and seeing the table and querying it via SQL
-  another example consuming the data programmatically in a way that points to the catalog files in S3.  E.g. a Spark SQL user setting up their connection to the iceberg data by pointing to it directly, presumably by setting the LOCATION attribute. Some guidance is here: https://iceberg.apache.org/docs/latest/spark-ddl/. but @jcipar  probably has the exact code required